src/common/test/main.cpp

```diff
@@ -21,12 +21,16 @@
 // |  DEALINGS IN THE SOFTWARE.                                                  |
```

Comment:
I think the diff on this file got screwed up, confuses version.hpp with main.cpp